sensor/tests/connection/connection_test.go

```diff
@@ -125,8 +126,16 @@ func Test_ScannerDefinitionsProxy(t *testing.T) {
 		req, err := http.NewRequest(http.MethodGet, "https://localhost:8443/scanner/definitions", nil)
 		require.NoError(t, err)
 
-		resp, err := client.Do(req)
+		// Sensor may not be yet in the ready state and may reject the API call
+		err = retry.WithRetry(func() error {
+			t.Logf("Attempting GET call to https://localhost:8443/scanner/definitions")
+			resp, err := client.Do(req)
+			if err != nil {
+				return err
+			}
+			require.Equal(t, http.StatusOK, resp.StatusCode, "response: %v", resp)
+			return nil
+		}, retry.WithExponentialBackoff(), retry.Tries(5))
```

Comment:
`WithExponentialBackoff` starts at 100ms wait and doubles each time, so this would run for 1.5s of waiting + the execution time of the GET call. Are you sure this is enough time for Sensor to reach the ready state or should we wait longer?

Reply:
This is a very good point - that 1.5s feels small and I am not sure what is the proper number here, but my gut-feeling would rather aim at something between 5 and 10s.

Reply:
I initially pushed [c29e288](https://github.com/stackrox/stackrox/pull/13532/commits/c29e2883793c9b35898bda2c748670d2c8e4be0f) to address this, but then rewritten the retry to use `assert.EventuallyWithT`. That should do the job as well.

Reply:
Looks good